DEPS

```diff
@@ -28,13 +28,79 @@ vars = {
   # dependencies of Dart are also updated to match the entries in the
   # Dart SDK's DEPS file for that revision of Dart. The DEPS file for
   # Dart is: https://github.com/dart-lang/sdk/blob/master/DEPS
-  'dart_revision': '65a5707189dec6c5e0174cbc821acfb90f0c9b3f',
+  'dart_revision': '4f9b25e410ed63028945e47386b434ed8cd8c313',
   # Dart calls the next one "boringssl_gen_rev"
   'dart_boringssl_gen_revision': '753224969dbe43dad29343146529727b5066c0f3',
   # Dart calls the next one "boringssl_rev"
   'dart_boringssl_revision': 'd519bf6be0b447fb80fbc539d4bff4479b5482a2',
   # Dart calls the next one "observatory_pub_packages_rev"
   'dart_observatory_packages_revision': '26aad88f1c1915d39bbcbff3cad589e2402fdcf1',
+  # Dart calls the next one "args_tag"
```

Comment:
I added a script in https://dart-review.googlesource.com/c/2200/ and renamed dart dependencies for ease of name-matching in 9e595bc.